dvc/repo/open_repo.py

```diff
@@ -67,6 +67,7 @@ def open_repo(url, *args, **kwargs):
         url = os.getcwd()
 
     if os.path.exists(url):
+        url = os.path.abspath(url)
```

Comment:
This was previously broken in the case where you passed a relative local path like `repo='.'` into `dvc.api` calls.